### PR TITLE
DOCS-10406 Jira Connector 1.0

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

File: jira/1.0/modules/ROOT/pages/jira-connector-studio.adoc [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+To add an operation for Jira Connector, follow these steps:
+
+. In the *Mule Palette* view, select *Jira* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials that the connector requires to access the target Jira system. Jira Connector supports ??? authentication methods.
```
Comment (devdanco) [inline jira/1.0/modules/ROOT/pages/jira-connector-studio.adoc]: Why are here 3 question marks ? Maybe typo

**Review by devdanco — DISMISSED** [review]

File: jira/1.0/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+* Creating a Mule app using Anypoint Studio (Studio)
+
+Before creating an app, you must have:
+
+* Credentials to access the target Jira resource
+* Anypoint Studio version 7.3.5 or later
+
+== Common Use Cases for the Connector
+
+Automate business processes by integrating Jira with other IT tools and systems in your enterprise such as messaging, CRM, messaging, and others.
```
Comment (varunmaker) [inline jira/1.0/modules/ROOT/pages/index.adoc]: @valkyrie69 it says messaging twice. Can we change one of them to "customer service"?

File: jira/1.0/modules/ROOT/pages/jira-connector-studio.adoc (hunk 1 above)
Comment (valkyrie69) [reply jira/1.0/modules/ROOT/pages/jira-connector-studio.adoc]: Placeholder. I didn't know the authentication methods until I got the reference. Thanks!

File: jira/1.0/antora.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name: jira-connector
+version: '1.0'
+display_version: 1.0 (Mule 4)
+title: Jira Connector
+nav:
+- modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    page-component-desc: Enables...
```
Comment (dhirsch1) [inline jira/1.0/antora.yml]: Need a description here

File: jira/1.0/modules/ROOT/pages/jira-connector-studio.adoc [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials that the connector requires to access the target Jira system. Jira Connector supports Basic and OAuth 2 authentication methods.  
+
+To configure the global element for Jira Connector, follow these steps:
+
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, On the *General* tab, in *Connection*, select the authentication method to configure:
```
Comment (dhirsch1) [inline jira/1.0/modules/ROOT/pages/jira-connector-studio.adoc]: Remove one "On the *General* tab"

File: jira/1.0/modules/ROOT/pages/jira-connector-xml-maven.adoc [hunk 5]
```diff
@@ -0,0 +1,72 @@
+= Microsoft Teams - XML and Maven Support - Mule 4
+
+Although you can manually code a Mule app in XML, it is more efficient to use Anypoint Studio:
```
Comment (dhirsch1) [inline jira/1.0/modules/ROOT/pages/jira-connector-xml-maven.adoc:3]: Remove this sentence.

File: jira/1.0/modules/ROOT/pages/jira-connector-xml-maven.adoc [hunk 6]
```diff
@@ -0,0 +1,72 @@
+= Microsoft Teams - XML and Maven Support - Mule 4
```
Comment (dhirsch1) [inline jira/1.0/modules/ROOT/pages/jira-connector-xml-maven.adoc]: Remove Mule 4

File: jira/1.0/modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -0,0 +1,52 @@
+= Jira Connector 1.0 - Mule 4
```
Comment (dhirsch1) [inline jira/1.0/modules/ROOT/pages/index.adoc]: Remove Mule 4

File: jira/1.0/modules/ROOT/pages/jira-connector-xml-maven.adoc (hunk 5 above)
Comment (valkyrie69) [reply jira/1.0/modules/ROOT/pages/jira-connector-xml-maven.adoc:3]: This sentence is in the template.

Comment (dhirsch1) [reply jira/1.0/modules/ROOT/pages/jira-connector-xml-maven.adoc:3]: I wonder why I thought it was out of place.

**Review by dhirsch1 — APPROVED** [review]

**Review by sathya0 — APPROVED** [review]

**Pull request merged**